spyder/plugins/application/widgets/install.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Update installation widgets."""
+
+# Standard library imports
+import logging
+import os
+import subprocess
+import tempfile
+import threading
+from urllib.request import urlretrieve
+
+# Third-party imports
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtWidgets import (QDialog, QHBoxLayout, QMessageBox,
+                            QLabel, QProgressBar, QPushButton, QVBoxLayout,
+                            QWidget)
+
+# Local imports
+from spyder import __version__
+from spyder.api.translations import get_translation
+from spyder.utils.icon_manager import ima
+from spyder.utils.programs import is_module_installed
+
+logger = logging.getLogger(__name__)
+
+# Localization
+_ = get_translation('spyder')
+
+# Update installation process statuses
+NO_STATUS = __version__
+DOWNLOADING_INSTALLER = _("Downloading installer")
+INSTALLING = _("Installing")
+FINISHED = _("Installation finished")
+PENDING = _("Pending update")
```

Comment:
Lets change these values following the suggestions from @CAM-Gerlach:
```suggestion
DOWNLOADING_INSTALLER = _("Downloading update")
INSTALLING = _("Installing update")
FINISHED = _("Installation finished")
PENDING = _("Update available")
```